cmd/kubeadm/app/cmd/phases/bootstraptoken.go

```diff
@@ -97,7 +97,7 @@ func NewCmdBootstrapToken() *cobra.Command {
 		Aliases: []string{"bootstraptoken"},
 	}
 
-	cmd.PersistentFlags().StringVar(&kubeConfigFile, "kubeconfig", "/etc/kubernetes/admin.conf", "The KubeConfig file to use when talking to the cluster")
+	cmd.PersistentFlags().StringVar(&kubeConfigFile, "kubeconfig", "/etc/kubernetes/admin.conf", "The kubeconfig file to use when talking to the cluster")
```

Comment:
same with KubeConfig.

Reply:
@timothysc, @fabriziopandini mentioned above that `kubeconfig` should be lower case:

> kubeadm or kubeconfig are not api objects, so we have to use small caps.

maybe the same applies to `Conformance` i have no information if that's an API object?

Reply:
It's a keyword reference in the e2e tests, and it's capitalized.  I'd verify by looking through the docs and being consistent there.

Reply:
odd, there seems to be a small contradiction between @fabriziopandini and @timothysc .
i will make the changes as @timothysc requests.